lib/jasmine.karma.worker.js

```diff
@@ -39,7 +39,11 @@
 
     const execute = async function () {
         await jasmineEnv.execute();
-        self.postMessage({event: "reportRunnerResults"});
+        // This doesn't actually wait for jasmine to finish.
+        // I don't know a way to run something when jasmine finishes running tests.
+        // Commenting this for now because it runs before the tests are done
+        // and results in karma stopping the headless browser.
+        // self.postMessage({event: "reportRunnerResults"});
```

Comment:
Why you just don't remove the line, committing with the message from comment? We can revert it anytime later.